src/plugins/recordTypes/organization/forms/default.jsx

```diff
@@ -0,0 +1,90 @@
+const template = (pluginContext) => {
+  const {
+    React,
+  } = pluginContext.lib;
+
+  const {
+    Panel,
+    Row,
+    Cols,
+    Col,
+  } = pluginContext.layoutComponents;
+
+  const {
+    Field,
+    InputTable,
+    Subrecord,
+  } = pluginContext.recordComponents;
+
+  return (
+    <Field name="document">
+      <Panel name="info" collapsible>
+        <Field name="orgTermGroupList">
+          <Field name="orgTermGroup">
+            <Panel>
+              <Row>
+                <Field name="termDisplayName" />
+                <Field name="termName" />
+                <Field name="termQualifier" />
+                <Field name="termStatus" />
+              </Row>
+
+              <Row>
+                <Field name="termType" />
+                <Field name="termFlag" />
+                <Field name="termLanguage" />
+                <Field name="termPrefForLang" />
+              </Row>
+
+              <InputTable name="nameDetail">
+                <Field name="mainBodyName" />
+                <Field name="additionsToName" />
+              </InputTable>
+
+              <InputTable name="termSource">
+                <Field name="termSource" />
+                <Field name="termSourceDetail" />
+                <Field name="termSourceID" />
+                <Field name="termSourceNote" />
+              </InputTable>
+            </Panel>
+          </Field>
+        </Field>
```

Comment:
It looks like we removed a bunch of fields from the term group in 4.1. Let's remove the same fields in 5.0, but reorganize a bit. Maybe

```
display name | term type     | status
language     | pref for lang | [blank column]
source fields
```